### ISSUE TITLE
npm 5.2 breaks Create React App

### ISSUE THREAD

**Issue by noccer** [npm/npm#17754]
#### I'm opening this issue because:

  - [x] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [ ] Other (_see below for feature requests_):

#### What's going wrong?
I am trying to use the latest version of [Create React App (1.3.3)](https://www.npmjs.com/package/create-react-app). After creating a new app using CRA1.3.3, I try to run `npm run start` but I get the following error:
```
./src/index.js
Module build failed: Error: Failed to load plugin import: Cannot find module 'eslint-plugin-import'
Referenced from: 
    at Array.forEach (native)
    at Array.reduceRight (native)
```

#### How can the CLI team reproduce the problem?

use latest npm 5.2

```
npm install -g create-react-app@latest
create-react-app example
cd example && npm run start
```
If you revert back to npm 5.1 (or presumably lower), it works absolutely fine.

I have tried earlier versions of Create React App (1.3.0, 1.2.0) with NPM 5.2, still the same errors. If I roll back to 5.1, no errors.

### supporting information:

 - `npm -v` prints: `5.2.0`
 - `node -v` prints: `v8.1.2`
 - `npm config get registry` prints: `https://registry.npmjs.org/`
 - Windows, OS X/macOS, or Linux?: MacOS 5.12.5
 - Network issues:
   - Geographic location where npm was run:
   - [ ] I use a proxy to connect to the npm registry.
   - [ ] I use a proxy to connect to the web.
   - [ ] I use a proxy when downloading Git repos.
   - [ ] I access the npm registry via a VPN
   - [ ] I don't use a proxy, but have limited or unreliable internet access.
 - Container:
   - [ ] I develop using Vagrant on Windows.
   - [ ] I develop using Vagrant on OS X or Linux.
   - [ ] I develop / deploy using Docker.
   - [ ] I deploy to a PaaS (Triton, Heroku).

**Comment by mtliendo** [comment]
I thought I was going crazy.  Literally 4 hours of debugging and I finally decided to google search:past 24 hours.  Thanks for posting! I came across the exact same issue.

**Comment by LitoMore** [comment]
Related to #17717

**Comment by ctrlplusb** [comment]
Related #17755

**Comment by kachkaev** [comment]
All worked for me after upgrading to npm 5.3. Can anyone else confirm please?

**Comment by noccer** [comment]
Resolved in npm@5.3.0+

#17755
